### PR TITLE
Don't put integration test data outside of test folder

### PR DESCRIPTION
**Pull request by mrsdizzie**
Some integration test data is being put outside of the integration test folders which means it isn't deleted properly with make clean. Update config so it works as expected.

Found 48GB of LFS test data and 17GB of attachments uncleaned in my data folder 😑

### CODE REVIEW

**Review by techknowlogick — APPROVED** [review]

File: integrations/mssql.ini.tmpl [hunk 1]
```diff
@@ -33,12 +33,15 @@ SSH_LISTEN_HOST  = localhost
 SSH_PORT         = 2201
 START_SSH_SERVER = true
 LFS_START_SERVER = true
-LFS_CONTENT_PATH = data/lfs-mssql
+LFS_CONTENT_PATH = integrations/gitea-integration-mssql/data/lfs-mssql
```
Comment (guillep2k) [inline integrations/mssql.ini.tmpl:36]:
```suggestion
LFS_CONTENT_PATH = {{REPO_TEST_DIR}}integrations/gitea-integration-mssql/data/lfs-mssql
```

File: integrations/mssql.ini.tmpl [hunk 2]
```diff
@@ -33,12 +33,15 @@ SSH_LISTEN_HOST  = localhost
...
 OFFLINE_MODE     = false
 LFS_JWT_SECRET   = Tv_MjmZuHqpIY6GFl12ebgkRAMt4RlWt0v4EHKSXO0w
 APP_DATA_PATH    = integrations/gitea-integration-mssql/data
 BUILTIN_SSH_SERVER_USER = git
 
+[attachment]
+PATH = integrations/gitea-integration-mssql/data
```
Comment (guillep2k) [inline integrations/mssql.ini.tmpl:43]:
```suggestion
PATH = {{REPO_TEST_DIR}}integrations/gitea-integration-mssql/data
```

File: integrations/mssql.ini.tmpl [hunk 3]
```diff
@@ -56,12 +59,14 @@ NO_REPLY_ADDRESS                  = noreply.example.org
 ENABLE_NOTIFY_MAIL                = true
 
 [picture]
-DISABLE_GRAVATAR        = false
-ENABLE_FEDERATED_AVATAR = false
+DISABLE_GRAVATAR              = false
+ENABLE_FEDERATED_AVATAR       = false
+AVATAR_UPLOAD_PATH            = integrations/gitea-integration-mssql/data/avatars
```
Comment (guillep2k) [inline integrations/mssql.ini.tmpl:64]:
```suggestion
AVATAR_UPLOAD_PATH            = {{REPO_TEST_DIR}}integrations/gitea-integration-mssql/data/avatars
```

File: integrations/mssql.ini.tmpl [hunk 4]
```diff
@@ -56,12 +59,14 @@ NO_REPLY_ADDRESS                  = noreply.example.org
...
+REPOSITORY_AVATAR_UPLOAD_PATH = integrations/gitea-integration-mssql/data/repo-avatars
```
Comment (guillep2k) [inline integrations/mssql.ini.tmpl:65]:
```suggestion
REPOSITORY_AVATAR_UPLOAD_PATH = {{REPO_TEST_DIR}}integrations/gitea-integration-mssql/data/repo-avatars
```

File: integrations/mssql.ini.tmpl [hunk 5]
```diff
@@ -56,12 +59,14 @@ NO_REPLY_ADDRESS                  = noreply.example.org
...
 
 [session]
-PROVIDER = file
-PROVIDER_CONFIG = data/sessions-mssql
+PROVIDER        = file
+PROVIDER_CONFIG = integrations/gitea-integration-mssql/data/sessions
```
Comment (guillep2k) [inline integrations/mssql.ini.tmpl:69]:
```suggestion
PROVIDER_CONFIG = {{REPO_TEST_DIR}}integrations/gitea-integration-mssql/data/sessions
```

**Review by guillep2k — CHANGES_REQUESTED** [review]
Please use the `{{REPO_TEST_DIR}}` prefix in all `.tmpl` files.

**Review by guillep2k — APPROVED** [review]

**Review by 6543 — APPROVED** [review]
Test PASS local too

**Comment by techknowlogick** [comment]
ping LG-TM

**Pull request merged**